frontend/src/components/pages/RequestKit.jsx

```diff
@@ -29,6 +30,24 @@ function RequestKit({ user }) {
     window.scrollTo(0, 0);
   }, []);
 
+  useEffect(() => {
+    const fetchKitName = async () => {
+      if (kitId) {
+        try {
+          const response = await fetch(`${API_URL}/kits/${kitId}`);
+          const data = await response.json();
+          setKitName(data.name);
+        } catch (error) {
+          console.error('Error fetching kit name:', error);
+        }
+      } else {
+        setKitName('');
+      }
+    };
+
+    fetchKitName();
+  }, [kitId]);
+
```

Comment:
The page `/kits` retrieves a list of all kits already, which means we already have a name of the kit, too. Please pass it as a prop instead of making a new request

Reply:
Done.